static/scss/partials/onboarding.scss

```diff
@@ -1,5 +1,6 @@
 // Protocol
 @import "libs/protocol/css/templates/multi-column.scss";
+@import "libs/protocol/css/components/zap.scss";
```

Comment:
Adding in [zap component](https://protocol.mozilla.org/patterns/atoms/zap.html) from Protocol.

Reply:
_(That's also all the zap.svgs)_